### PR TITLE
Fixed nav bar not responsive on landing page.

### PR DESCRIPTION
**Pull request by akhil-rana**
1. The navbar on the landing page was not responsive and all the content suddenly disappeared when the page is accessed from a smaller screen.
2. The content of the navbar was not of same font and size on landing and slides page and that looks a little uneven when switching from one page to other.
3. Signup option was not present on the slides page.

### CODE REVIEW

**Comment by akhil-rana** [comment]
before1


after1


The first image was before when the navbar was not reponsive.

File: apps/table.html [hunk 1]
```diff
@@ -72,6 +72,9 @@
 				</li>
 			</ul>
 			<ul class="navbar-nav">
+				<li class="nav-item link" style="font-family: sans-serif;">
+					<a class="nav-link" href="./signup/signup.html">Signup</a>
+				</li>
```
Comment (birm) [inline apps/table.html]: I'm a little divided on if table needs a signup page, but I think it makes sense for consistency.

**Comment by akhil-rana** [comment]
ezgif-3-c799524c2d6f

As can be seen that the navbar height, font size and style was not constant for both the pages. Also the signup option was not present on the slides page.

File: apps/landing/landing.html [hunk 2]
```diff
@@ -6,14 +6,22 @@
...
     <link rel="stylesheet" href="./main.css" />
     <link rel="shortcut icon" type="image/x-icon" href="favicon.png">
+    <script src="https://code.jquery.com/jquery-3.4.1.slim.min.js"
+		integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n"
+		crossorigin="anonymous"></script>
+	<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"
+		integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6"
+		crossorigin="anonymous"></script>
+	<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
+		integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
```
Comment (birm) [inline apps/landing/landing.html]: For future-readiness, I'll invite @nanli-emory to share thoughts on bootstrap on the landing page.

Comment (akhil-rana) [reply apps/landing/landing.html]: Okay. Thanks. 😀

Comment (nanli-emory) [reply apps/landing/landing.html]: Pretty good.

File: apps/table.html (hunk 1 above)
Comment (nanli-emory) [reply apps/table.html]: We should have a discussion about the registration process before working on the `signup` page.

**Pull request merged**